ganache-with-protocol/build-ganache.sh

```diff
@@ -0,0 +1,90 @@
+git clone -b streamflow https://github.com/livepeer/protocol.git
+cd protocol
+echo "Setting devenv specific protocol parameters"
+migrations="/protocol/migrations/migrations.config.js"
+sed -i 's/roundLength:.*$/roundLength: 50,/' $migrations
+sed -i 's/unlockPeriod:.*$/unlockPeriod: 50,/' $migrations
+sed -i 's/numActiveTranscoders:.*$/numActiveTranscoders: 50,/' $migrations
+sed -i 's/numTranscoders:.*$/numTranscoders: 100,/' $migrations
+
+cat << EOF > /protocol/truffle.js
+require("babel-register")
+require("babel-polyfill")
+
+module.exports = {
+    networks: {
+        development: {
```

Comment:
If we use this development network then the `AdjustableRoundsManager` will be deployed instead of the `RoundsManager` (https://github.com/livepeer/protocol/blob/d6bcb03337839071f1a90e391760ef1f7660f836/migrations/3_deploy_contracts.js#L43). If we use the lpTestNet network (similar to what is done in [geth-with-protocol](https://github.com/livepeer/docker-livepeer/blob/8e5c1757aadad5f460dee91fe922acffc4b5ca32/geth-with-protocol/build-protocol.sh#L25)) then the `RoundsManager` will be deployed. I think @adamsoffer has been writing his tests based on the `RoundsManager` being deployed.

Reply:
Yeah, we need to use the `RoundsManager` for the subgraph integration tests.

Reply:
fixed in 25e0ca1